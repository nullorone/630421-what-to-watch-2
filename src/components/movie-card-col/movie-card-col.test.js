import React from "react";
import renderer from "react-test-renderer";
import MovieCardCol from "./movie-card-col";

describe(`Make snapshot`, () => {
  const initProps = {
    type: `academy`,
  };

  it(`Get snapshot component`, () => {
    const tree = renderer
      .create(
          <MovieCardCol {...initProps}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
