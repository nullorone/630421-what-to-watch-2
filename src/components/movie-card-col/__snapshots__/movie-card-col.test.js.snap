// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make snapshot Get snapshot component 1`] = `
<div
  className="movie-card__academy-col"
/>
`;
